src/hb-ot-color-colr-table.hh

```diff
@@ -970,13 +974,20 @@ struct COLR
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
+#if COLRV1_DISABLE_SUBSETTING == 1
+    return_trace (c->check_struct (this) &&
+                  (this+baseGlyphsZ).sanitize (c, numBaseGlyphs) &&
+                  (this+layersZ).sanitize (c, numLayers) &&
+                  (version == 0));
+#else
```

Comment:
Can you rewrite to avoid duplication? And, flip the macro name:
```
  bool sanitize (hb_sanitize_context_t *c) const
  {
    TRACE_SANITIZE (this);
    return_trace (c->check_struct (this) &&
                  (this+baseGlyphsZ).sanitize (c, numBaseGlyphs) &&
                  (this+layersZ).sanitize (c, numLayers) &&
                  (version == 0
#ifdef HB_ENABLE_COLRV1
                                 || (version == 1 &&
                                    baseGlyphsV1List.sanitize (c, this) &&
                                    layersV1.sanitize (c, this) &&
                                    varStore.sanitize (c, this))
#2957 
               ));
  }
```

Reply:
Done